crates/cheatnet/src/cheatcodes/deploy.rs

```diff
@@ -77,6 +78,19 @@ impl CheatnetState {
                     ))))
                 }),
             CallContractOutput::Panic { panic_data } => {
+                let err_str = as_cairo_short_string(&panic_data[0]).unwrap();
```

Comment:
Should we check the first element only?

Reply:
Maybe we should iterate the whole array so as not to miss anything

Reply:
In the case, there are multiple error messages in the panic data